.github/workflows/pr_ci_frontend.yaml

```diff
@@ -1,29 +1,35 @@
-name: pr_frontend_formatting_check
-on:
-  pull_request_target:
-    branches:
-      - main
-    types: [opened, reopened, synchronize]
-  workflow_run:
-    workflows: [pr_maintainer_checklist]
-    types:
-      - completed
-
-jobs:
-  prettier:
-    name: Run PR Frontend Formatting Check
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Setup Node
-        uses: actions/setup-node@v3
-
-      - name: Install Prettier
-        run: npm install prettier
-
-      - name: Run Prettier
-        working-directory: ./frontend
-        run: |
-          npx prettier . --check --config ../.prettierrc --ignore-path ../.prettierignore
+name: pr_ci_frontend
+on:
+  pull_request_target:
+    branches:
+      - main
+    types: [opened, reopened, synchronize]
+  workflow_run:
+    workflows: [pr_maintainer_checklist]
+    types:
+      - completed
+
+jobs:
+  frontend:
+    name: Run PR Frontend Formatting Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+
+      - name: Yarn Install
+        uses: borales/actions-yarn@v4
+        with:
+          cmd: install
+          dir: "frontend"
+
+      - name: Install Prettier
+        run: yarn add prettier
+
+      - name: Run Prettier
+        working-directory: ./frontend
+        run: |
+          yarn prettier . --check --config ../.prettierrc --ignore-path ../.prettierignore
```

Comment:
Looks good, @CaioCintra and it works 👍!  

But we also wanted to **consolidate/combine** the `pr_ts_typecheck` with `pr_frontend_formatting_check`.

To consolidate the two workflows you need to add `contine-on-error: true` to prettier step and below that add the typecheck step. Finally we can delete the `pr_ts_typecheck.yaml` file.
```yaml
  - name: Run Prettier
    working-directory: ./frontend
    run: |
      yarn prettier . --check --config ../.prettierrc --ignore-path ../.prettierignore
    continue-on-error: true
      
 - name: Run Type Check
    uses: borales/actions-yarn@v4
    with:
      cmd: nuxi typecheck
      dir: "frontend"
```

Reply:
Ok, just adding this to frontend and removing ts file, right? I'm sending another PR

Reply:
I think I messed something out, is not working anymore

Reply:
No, worries. I will give the workflow a try locally now and the checks failed due to other reason 😄

Reply:
Hey @CaioCintra! No stress on the workflow errors. Looks like @to-sta and I might have introduced some errors in a large rewrite we did earlier today. We'll check this soon and get back to you!

Reply:
@to-sta: a3bb23a has `yarn nuxi typecheck` up and running again :) We had just missed adding some indexes.

Reply:
Ok, thanks

Reply:
One issue that is preventing the workflows from working is the indentation. 

```yaml
 - name: Run Type Check
    uses: borales/actions-yarn@v4
    with:
      cmd: nuxi typecheck
      dir: "frontend"

# Should look like this
  - name: Run Type Check
    uses: borales/actions-yarn@v4
    with:
      cmd: nuxi typecheck
      dir: "frontend" 
```
In case you are using VScode i can recommend the [GitHub Actions](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions) extension. It will help you with issues like that.